lib/themes/dosomething/paraneue_dosomething/js/components/FeatureCard.js

```diff
@@ -44,9 +45,13 @@ class FeatureCard extends React.Component {
     this.campaignsApiPage = 1;
 
     this.state = {
+      actionDisabled: false,
       campaignIndex: -1,
-      isSignedUp: ''
+      isSignedUp: false,
+      isLoading: false,
     };
+
+    this.signup = this.signup.bind(this);
```

Comment:
wait this works? I just tried it in both a node console & browser window, both threw an error for bind of undefined. (am I not seeing where this is defined beforehand?)

couldn't this just be `this.signup = this;`? bind feels unnecessary here